### PR TITLE
Deconfigures indexing for clarity.

### PR DESCRIPTION
**Pull request by justinlittman**
Following the approach set out in #96, switched from indexing by configuration to cleaner, simpler code.

Also, removed autosuggest since not used.

### CODE REVIEW

File: src/Indexer.js [hunk 1]
```diff
@@ -15,6 +15,10 @@ export default class Indexer {
     this.logger = new Logger()
     this.knownIndexResults = ['created', 'updated']
     this.knownDeleteResults = ['deleted']
+    this.indexers = {
+      sinopia_templates: SinopiaTemplateIndexer,
+      sinopia_resources : ResourceIndexer
```
Comment (mjgiarlo) [inline src/Indexer.js]: Stray space after `sinopia_resources` that I would have expected `eslint` to catch. No biggie.

File: config/default.js [hunk 2]
```diff
@@ -15,67 +15,6 @@ module.exports = {
   defaultMimeType: process.env.DEFAULT_MIME_TYPE || 'application/ld+json',
   indexType: process.env.INDEX_TYPE || 'sinopia',
   indexUrl: process.env.INDEX_URL || 'http://localhost:9200',
-  // Note that InputLookupSinopia expects uri and label fields.
-  indexMappings: process.env.INDEX_FIELD_MAPPINGS
```
Comment (mjgiarlo) [inline config/default.js]: Looks like there's some code in `search.js` that may still use the old name of this config value (`indexFieldMappings`). Could possibly be done in a separate commit or PR.

Comment (justinlittman) [reply config/default.js]: I removed search.js. Given how easy it is to perform a query with curl this seems unnecessary and more likely to confuse, as it does not accurately reflect the queries that are performed by the editor.

Comment (mjgiarlo) [reply config/default.js]: :clinking_glasses:
